data/speakers.yml

```diff
@@ -5913,13 +5913,6 @@
   website: ""
   slug: james-duncan-davidson
   canonical_slug:
-- name: James Edward Gray
-  twitter: ""
-  github: JEG2
-  bio: ""
-  website: http://graysoftinc.com/
-  slug: james-edward-gray
-  canonical_slug:
```

Comment:
Let's keep this one too and set the `canonical_slug` to `james-edward-gray-ii` so future references point to the right record.